pkgs/development/python-modules/sphinxcontrib-bayesnet/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchPypi, sphinx, sphinxcontrib-tikz }:
+
+buildPythonPackage rec {
+  pname = "sphinxcontrib-bayesnet";
+  version = "0.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0x1kisvj7221cxfzmwplx3xlwbavl636fpncnjh7gghp1af71clw";
+  };
+
+  propagatedBuildInputs = [ sphinx sphinxcontrib-tikz ];
+
+  # No tests
+  doCheck = false;
+  pythonImportsCheck = [ "sphinxcontrib.bayesnet" ];
```

Comment:
Does the namespaced package require special handling? /cc @jonringer

Reply:
Can't see anything special in the other similar packages:
- sphinxcontrib-bibtex
- sphinxcontrib-autoapi
- sphinxcontrib-blockdiag
To name a few. (Didn't check all.)